### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Jaserelhabrouk**
The compilation part I'm not sure of what exactly to write

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+# Carleton University
+    Authors
+        -Mahsa Layeghi
+        -Parichehreh Firoozishahmirzadi
+        -Jin Xu
+        -Jaser El-Habrouk
+
+The application is for fun purposes where the player has to reach a certain goal on a map while trying to avoid barriers and arrowns.
+
+# Folder Organization
```
Comment (Pari-Firoozi) [inline README.md:13]: There are some other folders such as : bin, build, data, makefile, .... as well.
Please  include those and describe the content as well.

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+/src folder contains all the functions needed for the game as well as the main function.
+/test folder contains all the test functions for the game functions.
+/include has all the include files needed for the application's functions
+
+# Compilation
```
Comment (Pari-Firoozi) [inline README.md:18]: The reader need to compile the whole program completely using the information provided in readme. This does not have this information.
Please check Makefile to see how to compile and run the program.
Also, you need to mention the program dependencies(SDL library) and a link to wiki.

**Review by Pari-Firoozi — CHANGES_REQUESTED** [review]
Please fix the requirments and update it ASAP.

File: README.md (hunk 1 above)
Comment (Jaserelhabrouk) [reply README.md:13]: bin build and some files does not have anything in them so I can't describe any content

**Pull request merged**